packages/kit/src/runtime/server/index.js

```diff
@@ -364,6 +364,9 @@ export async function respond(request, options, state) {
 		if (error instanceof Redirect) {
 			return redirect_response(error.status, error.location);
 		}
+		if (error instanceof HttpError) {
+			return static_error_page(options, error.status, error.body.message);
```

Comment:
I'm confused why this is necessary. `handle_fata_error` wraps this functionality, so the outcome should be the same

Reply:
It seems some behaviour has changed since last week. I have updated the example: https://stackblitz.com/edit/sveltejs-kit-template-default-djcepj?file=src/hooks.server.ts

When I opened the issue last week, I got a 500 error when **hovering** over the link.
With the latest `SvelteKit` version hovering works, but **clicks** result in a 500 error shown.

Reply:
sounds related to #8108

Reply:
I think those are two different issues. I could not reproduce the issue described in #8108